packages/pigeon/platform_tests/android_kotlin_unit_tests/pubspec.yaml

```diff
@@ -0,0 +1,20 @@
+name: android_kotlin_unit_tests
+description: Unit tests for Pigeon generated Java code.
+
+publish_to: "none" # Remove this line if you wish to publish to pub.dev
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.7.0 <3.0.0"
```

Comment:
We're working on eliminating use of pre-NNBD anywhere in the repository, so this should be 2.12+

Reply:
Updated